wwwroot/buy-anything.html

```diff
@@ -0,0 +1,414 @@
+                                        <br> d) It is all legal to do so.
+                                        <br>
+                                        <br> An offer will be issued only until these 4 points are sorted out. If it’s not possible, you will receive a friendly decline with an explanation of why.
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq4">
+                                <label for="faq4">
+                                    4. How is the bitcoin price calculated at the time of the transaction?
+                                </label>
+                                <div class="card-content">
+                                    <p>The offer quote is always in fiat (in USD precisely) and the final BTC amount is displayed when the user agrees to buy now. The exchange rate is reserved for 30 minutes.</p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq5">
+                                <label for="faq5">
+                                    5. Will I get an invoice?
+                                </label>
+                                <div class="card-content">
+                                    <p>Yes, it will be available through a download link.</p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq6">
+                                <label for="faq6">
+                                    6. What about duties and taxes?
+                                </label>
+                                <div class="card-content">
+                                    <p>It will depend on the source and destination of the product. If it has to cross borders, it might be subject to duties or taxes. If it can be purchased in the same country as it’s being delivered, there will be no duty taxes added to the final price.</p>
+                                </div>
+                            </div>
+                            <div class="card">
+                                <input type="checkbox" id="faq7">
+                                <label for="faq7">
+                                    7. How long does it take to ship a physical product? A digital product?
+                                </label>
+                                <div class="card-content">
+                                    <p>YThere is no general answer to this question for physical products. It depends on where the product is (ShopinBit’s warehouse or third party warehouse) and how it will be shipped to you.
```

Comment:
```suggestion
                                    <p>There is no general answer to this question for physical products. It depends on where the product is (ShopinBit’s warehouse or third party warehouse) and how it will be shipped to you.
```